test/object-store/util/test_utils.cpp

```diff
@@ -70,12 +70,24 @@ void catch2_ensure_section_run_workaround(bool did_run_a_section, std::string se
     }
 }
 
-std::string encode_fake_jwt(const std::string& in)
+std::string encode_fake_jwt(const std::string& in, int64_t exp, int64_t iat)
 {
+    // by default make a valid expiry time so that the sync session pre check
+    // doesn't trigger a token refresh on first open
+    using namespace std::chrono_literals;
+    if (!exp) {
```

Comment:
Kind of confusing that `0` means "30 minutes from now", I guess? Consider changing `exp` and `iat` to be `Optional<int64_t>`.